exporters/autoexport/metrics_test.go

```diff
@@ -63,3 +72,24 @@ func TestMetricExporterOTLPOverInvalidProtocol(t *testing.T) {
 	_, err := NewMetricReader(context.Background())
 	assert.Error(t, err)
 }
+
+func TestMetricExporterPrometheus(t *testing.T) {
+	t.Setenv("OTEL_METRICS_EXPORTER", "prometheus")
```

Comment:
Can we add `defer goleak.VerifyNone(t)` to this test to make sure that we are not leaking any goroutine?

Reply:
Done.